### PR TITLE
Added Impact review to CrystalQuest

### PR DESCRIPTION
**Pull request by GingertronMk1**
Dunno how to do snapshots, if someone wants to tell me I'd be grateful

### CODE REVIEW

**Comment by wjdp** [comment]
@GingertronMk1 There's a section about how to do snapshots on this page https://history.newtheatre.org.uk/docs/link-list/

**Comment by GingertronMk1** [comment]
Ah, apologies, I've been sort of doing this based off other files

**Comment by wjdp** [comment]
No trouble :smile:

**Comment by GingertronMk1** [comment]
Updated

**Comment by BenWoodford76** [comment]
Looks good to me now. @wjdp anything glaringly obvious to you?

File: _shows/16_17/crystal_quest.md [hunk 1]
```diff
@@ -39,6 +39,17 @@ prod_shots: bjxVx7
...
+links:
+  - type: Review
+    href: http://www.impactnottingham.com/2017/06/crystal-quest-5d-adventure-nottingham-new-theatre/
+    snapshot: E7Lcm
+    publisher: Impact Nottingham
+    author: Emma Pallen
+    title: "Crystal Quest: A 5D Adventure @ Nottingham New Theatre"
+    date: 2017-06-15
+    rating: 10/10
+    quote: “Crystal Quest: A 5D Adventure is a one-of-a-kind production that can be watched again and again”
```
Comment (wjdp) [inline _shows/16_17/crystal_quest.md]: I'd take the title out of the quote, rather superfluous.

**Comment by wjdp** [comment]
Other than that, no. Now the preview works please check that.

File: _shows/16_17/crystal_quest.md (hunk 1 above)
Comment (GingertronMk1) [reply _shows/16_17/crystal_quest.md]: Good shout, I just took a pull quote from the article itself

**Comment by BenWoodford76** [comment]
I'm getting an issue... 
image

**Comment by BenWoodford76** [comment]
I have no clue why this is happening....

**Comment by wjdp** [comment]
Yeah, just noticed that. Some change to the file appears to have broken everything. Very odd.

**Comment by BenWoodford76** [comment]
awks

File: _shows/16_17/crystal_quest.md (hunk 1 above)
Comment (wjdp) [inline _shows/16_17/crystal_quest.md]: Actual problem: The quote marks you've used are incorrect. Unsure what editor you're using but stylised quote marks have snuck in: `“Crystal ... again”` should be `"Crystal ... again"`.

(Techincal background about quotemarks here https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html, the table at the top shows the various possible symbols)

Comment (GingertronMk1) [reply _shows/16_17/crystal_quest.md]: Ah, when I said I lifted the pull-quote from the Impact site, it came with the quote marks. Good to know for the future.

Comment (wjdp) [reply _shows/16_17/crystal_quest.md]: YAML issues tend to be a bit of a PITA. 

For reference: I shoved the frontmatter block (everything between the `---` marks, inclusive) into http://www.yamllint.com/ which points out where the issue is.

image

**Comment by BenWoodford76** [comment]
Deploy preview looks good, no glaring issues so am happy to merge

**Pull request merged**